### DISCUSSION TITLE
Add graphs to the /sites page

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#674] [Anything Else]
"it’s great to have the traffic of all sites at a glance and jump into those where something interesting seem to have happened. graphs help with that"

(feedback via twitter dm)

**Comment by marcbria** [comment]
+1

Not a must, but probably is something that will make plausible more fancy. ;-)
Are you talking about static graphs or showing realtime data?

In both cases sounds like it will make the /sites dashboard very heavy... so probably it need to be something to be enabled/disabled in the config settings.

**Comment by metmarkosaric** [comment]
**Update:** We’ve made some improvements to the “My Sites” page:

* You can now search for any of your sites
* There’s a graph for an overview of the traffic 
* And a comparison to the previous period so you can see the trend

Have fun exploring!
